cfgov/diversity_inclusion/jinja2/diversity_inclusion/voluntary-assessment-form.html

```diff
@@ -12,132 +12,217 @@
     TODO: Come back to this
 {%- endblock %}
 
-{% block og_image %}
-    TODO: Come back to this
-{% endblock %}
-
 {% block css -%}
     {{ super() }}
     <style>
 
     </style>
 {%- endblock %}
 
-{% block content_main -%}
-    <h1>Voluntary Diversity Assessment Registration Form</h1>
-
-    <form id="quiz-form" action="{{ url('voluntary_assessment_form') }}" method="POST">
-        {{ form }}
-        <input type="hidden" name="csrfmiddlewaretoken" value="{{ csrf_token }}">
-        <input type="submit" value="Submit">
-    </form>
-{%- endblock %}
-
-{% block content_sidebar -%}
-    <div class="block block__flush-top">
-        <h3>About us</h3>
-        <p>
-            We're the Consumer Financial Protection Bureau (CFPB),
-            a U.S. government agency that makes sure banks, lenders,
-            and other financial companies treat you fairly.
-        </p>
-        <a class="a-link a-link__jump" href="/about-us/the-bureau/">
-            <span class="a-link_text">Learn how the CFPB can help you</span>
-        </a>
+{% block pre_content %}
+    <div class="content_wrapper">
+        <nav class="breadcrumbs" aria-label="Breadcrumbs">
+            {{ svg_icon('left') }}
+            <a class="breadcrumbs_link" href="/about-us/diversity-and-inclusion/">
+                Diversity and inclusion at the Bureau
+            </a>
+        </nav>
     </div>
+{% endblock %}
 
-    <div class="block">
-        <header class="m-slug-header">
-            <h2 class="a-heading">About this questionnaire</h2>
-        </header>
+{% block content_main -%}
+    <div class="block block__flush-top">
+        <h1>Voluntary diversity assessment registration form</h1>
+        <div class="lead-paragraph">
+            <p>
+                The Consumer Financial Protection Bureau’s (Bureau) D&I Assessment website allows financial institutions to submit D&I Assessments.
```

Comment:
Diversity & Inclusion should be spelled out the first time here, right? Also, unless lawyers demand it, `(Bureau)` feels unnecessarily formal and of limited value in this setting.

Reply:
Fixed in 612d440